tools/dockerfile/bazel/Dockerfile

```diff
@@ -32,21 +32,20 @@ RUN apt-get update
 RUN apt-get -y install software-properties-common python-software-properties
```

Comment:
They could be moved to another PR, but they're necessary (`make_grpcio_tools.py` does the wrong thing with the dockerfile being broken).